inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -281,73 +274,92 @@ dat_raw <- data_path %>%
   mutate(across(where(\(x) inherits(x, "POSIXct")), as.Date))
 ```
 
-Once imported into __R__, the dataset called `dat` includes:
+Data used in this report _are available to the reader at https://doi.org/10.1038/s41597-020-0448-0 _, and contains the following variables:
 
-* `date`: the date of admission
-* `region`: the NHS region
-* `org_name`: the full name of the NHS trust
-* `org_code`: a short code for the NHS trust
-* `n`: number of new, confirmed COVID-19 cases admitted, including inpatients
-  who tested positive on that day, and new admissions with a positive test
+```{r}
+# This is what the data used in this report, `dat_raw`, looks like:
+knitr::kable(head(dat_raw),
+             align = "lccc"
```

Comment:
I added this argument because the headings of the columns were squashed together and it was difficult to read when printing the table
What other approach should I use to print the table as an output for the reader? maybe I could try emulating the way that other tables have been printed in the report?

Reply:
Yes, you will get better results using `kbl()`. E.g., 

```r
head(dat_raw) %>%
  kbl() %>%
  kable_styling()
```

Reply:
Added in 9f8064489ea51dd360874965f3d5e04aaf01b920